samples/nrf9160/serial_lte_modem/README.rst

```diff
@@ -0,0 +1,134 @@
+.. _serial_lte_modem	:
+
+nRF9160: Serial LTE Modem
+#########################
+
+The Serial LTE Modem (SLM) sample is an enhancement to at_client sample:
+ - support TCP/IP proprietary AT commands
+ - support GPS proprietary AT commands
+ - support communication to external MCU over UART
+
+All nRF91 modem AT commands are also supported.
+More proprietary AT commands could be added in the future.
+
+Requirements
+************
+
+* The following development board:
+
+    * nRF9160 DK board (PCA10090)
+
+* When client is PC
+
+    * Any terminal software, e.g. Teraterm
+
+* When client is nRF52
+
+    * nRF52840 DK board (PCA10056)
+    * nRF52832 DK board (PCA10040)
+
+By default CONFIG_SLM_CONNECT_UART_0 is defined, user can use the J-Link
+COM port on PC side to connect with nRF9160 DK, and send/receive AT
+command there.
+
+* Terminal serial configuration
+    * Hardware flow control: disabled
+    * Baud-rate: 115200
+    * Parity bit: no
+
+NOTE-1 the default AT command terminator is Carrier Return and Line Feed,
+i.e. "\r\n". NOTE-2 nRF91 logs is outputted to the same terminal port.
+
+To work with external MCU (nRF52), CONFIG_SLM_CONNECT_UART_2 is to be defined.
+The pin inter-connection between nRF91 and nRF52 is shown as in below table
+
+.. list-table::
+   :align: center
+
+   * - nRF52 DK
+     - nRF91 DK
+   * - UART TX P0.6
+     - UART RX P0.11
+   * - UART RX P0.8
+     - UART TX P0.10
+   * - UART CTS P0.7
+     - UART RTS P0.12
+   * - UART RTS P0.5
+     - UART RTS P0.13
+   * - GPIO OUT P0.27
+     - GPIO IN P0.31
+
+* UART instance in use
+    * nRF52840 nRF52832 (UART0)
+    * nRF9160 (UART2)
+
+* UART configuration
+    * Hardware flow control: enabled
+    * Baud-rate: 115200
+    * Parity bit: no
+    * Operation mode: IRQ
+
+Please NOTE the GPIO output level on nRF91 side should be 3V.
+
+TCP/IP AT Commands
+******************
+
+* AT#XSOCKET=<op>,<type>
+* AT%XSOCKET?
```

Comment:
Small typo: "AT%XSOCKET?" --> "AT#XSOCKET?"

Eagerly looking forward to this branch being finished and merged into master :-)